backend/models/users.js

```diff
@@ -3,10 +3,15 @@ var models = require('../models');
 var uuidv4 = require('uuid/v4');
 var storage = require('../utils/storage');
 
-module.exports.addUserToTripByUserId = (tripId, userId, callback) => {
+module.exports.addUserToTripByUserId = (tripId, userId, readOnly, callback) => {
+  var readOnlyPermission = false;
```

Comment:
You can make this `= readOnly ? true : false`

Reply:
done